test/wrappers/ortools_test.rb

```diff
@@ -2068,344 +2068,6 @@ def test_setup_duration
     assert_equal problem[:services].size, result[:routes][1][:activities].size - 1
   end
 
-  def test_pickup_delivery
```

Comment:
Why removing those tests ? We need to ensure this still works after our clean. These tests can be cleaned thought.

Reply:
Theses tests were verifying an unreachable part of the code